code/datums/extensions/extensions.dm

```diff
@@ -26,7 +26,7 @@
 		return STATUS_CLOSE
 	if(!all_predicates_true(list(user), user_predicates))
 		return STATUS_CLOSE
-	if(holder.CanUseTopic(usr, default_state) != STATUS_INTERACTIVE)
+	if(holder.CanUseTopic(usr,GLOB.default_state) != STATUS_INTERACTIVE)
```

Comment:
That seems it should be `SSnano.default_state` not `GLOB.default_state`

Reply:
It's the same thing, but you are right

Reply:
Actually now its not the same, it was moved to GLOB, i can change it to GLOB.nano_default_state for easy understanding but this will mess up porting in the future, so id rather have it this way